hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -840,6 +840,47 @@
     }
 
     // RSTMGR tests:
+    {
+      name: chip_rstrmgr_reset_info
+      desc: '''Verify the `reset_info` CSR register.
+
+            Generate all 6 types of reset, and check the reset_info is as expected.
```

Comment:
I think it is worth adding the following line to clarify that this is not going to be a new test being developed:
```
  This testpoint cross-references the individual IP tests that generate those reset requests. 
```

Reply:
Done